test/lib/eventFactory.test.js

```diff
@@ -508,11 +508,6 @@ describe('Event Factory', () => {
                         eventId: model.id,
                         jobId: 9
                     }));
-                    assert.calledWith(buildFactoryMock.create, sinon.match({
-                        parentBuildId: 12345,
-                        eventId: model.id,
-                        jobId: 10
-                    }));
```

Comment:
The value of trigger `startFrom` is `~pr:branch`, so `jobId 10` should not be selected.
https://github.com/screwdriver-cd/models/blob/master/test/lib/eventFactory.test.js#L306